.github/workflows/build.yml

```diff
@@ -15,30 +15,38 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
```

Comment:
Nice touch. After some point I would get rid of this either way (as 3.12 is close)

Reply:
Thank you.